runtime/appruntime/api/server.go

```diff
@@ -59,12 +59,18 @@ type Handler interface {
 	Handle(c IncomingContext)
 }
 
+type RequestsTotalLabels struct {
+	Service  string // Service name.
+	Endpoint string // Endpoint name.
+	Code     string // Human-readable HTTP status code.
```

Comment:
Should this be an int? Do we support that?

Reply:
`Code` is a string like `ok` or `not_found`. If we can't convert a HTTP code into a human-readable string, we set `Code` to `http_X` (e.g. `http_418`).